tensorboard/components_polymer3/BUILD

```diff
@@ -64,12 +64,23 @@ tf_ts_library(
 )
 
 tf_js_binary(
-    name = "polymer3_lib_binary",
+    name = "polymer3_lib_binary_no_shim",
     compile = True,
     entry_point = ":polymer3_lib.ts",
     deps = [":polymer3_ts_lib"],
 )
 
+genrule(
+    name = "polymer3_lib_binary",
+    srcs = [
+        # Do not sort. order is important.
+        "@npm//:node_modules/web-animations-js/web-animations-next-lite.min.js",
+        ":polymer3_lib_binary_no_shim.js",
+    ],
+    outs = ["polymer3_lib_binary.js"],
+    cmd = "for f in $(SRCS); do cat \"$$f\"; echo; done > $@",
```

Comment:
Hm, can you just do `cat $(SRCS) >$@`?

Reply:
There is a sourcemap at the end of the file and simple concatenation actually result in wrong code. I wanted new line delimited concatenation. Please advise on how to do that more cleanly.

Reply:
Ah I see, that's fine then, I don't know of a better way offhand.